tests/unit/testing/test_cos.py

```diff
@@ -26,6 +27,73 @@
     get_alert_rules,
 )
 
+GRAFANA_AGENT_METRICS_TARGETS = """
```

Comment:
Is there a chance this will change in the future ? If so what is the way to update this json?

Reply:
Potentially yes, but that would be major change, so ideally grafana-agent will be in new track channel. Not sure how we can do this more smartly, but I'm afraid we can do this only manually for such version.